rfcs/20230418-add-pow-graph-op/README.md

```diff
@@ -0,0 +1,267 @@
+|-----------|------------|-----------|----------|----------------|-----------------------|
+| input     | src        | src       | src      | src            | src                   |
+| input     | exponent   | exponent  | exponent | exponent       |                       |
+| attribute |            |           |          |                | exponent              |
+| attribute |            |           |          |                | alpha                 |
+| attribute |            |           |          | auto_broadcast |                       |
+| output    | dst        | dst       | dst      | dst            | dst                   |
+
+Most of them take `Pow` as a binary element-wise operation, while oneDNN uses it
+as an unary element-wise operation.
+
+Numpy broadcasting rule is supported for `Pow` in these frameworks and toolkit.
+And in Openvivo, when attribute `auto_broadcast` is `none`, shapes of `src` and
+`exponent` must be same.
+
+The src and dst tensors should have the same data type and support floating
+point data type (`f32`, `bf16` and `f16`). The exponent tensors can support
+floating point data type (`f32`, `f16` and `f16`) as well. Attribute is a
+single-value `f32` input.
+
+Dst shape equals to broadcasted shape of inputs.
+
+##### Rsqrt
+
+`Rsqrt` op computes the reciprocal of the square-root of each element of
+`input`. The src and dst tensors should have the same data type and support
+floating point data type(`f32`, `bf16` and `f16`).
+
+`Rsqrt` does the following computation with src:
+
+$$ dst_{i} = \frac{1}{\sqrt{src_{i}}} $$
+
+| Rsqrt     | Tensforflow | Pytorch | ONNX* | OpenVINO* | oneDNN* |
+|-----------|-------------|---------|-------|-----------|---------|
+| input     | src         | src     |       |           |         |
+| attribute |             |         |       |           |         |
+| attribute |             |         |       |           |         |
+| output    | dst         | dst     |       |           |         |
+
+*：ONNX, OpenVINO and oneDNN does not support Rsqrt directly, but in ONNX and
+OpenVINO Rsqrt can be supported by Pow(src, -1/2) and in oneDNN it is supported
+by Pow(src) with attribute -1/2.
+
+## Proposal
+
+To support the request from framework, we need to add new operations to graph
+API, with 5 options provides as follows:
+
+### Option 1: Add Unary Element-wise Pow only
+
+Add eltwise `Pow` that takes a single input tensor src, with exponent as an
+single-value attribute.
+`Rsqrt` reuses `Pow` with exponent attribute -1/2.
+
+| Pow       | Argument Name    | Required or Optional       | Data Type     |
+|-----------|------------------|----------------------------|---------------|
+| input     | `src`            | Required                   | f32,f16,bf16* |
+| attribute | `exponent`       | Required                   | f32           |
+| attribute | `auto_broadcast` | optional, default: 'numpy' | string        |
```

Comment:
For this option, what do you mean by supporting broadcasting?

Reply:
Good catch. Thanks you. `auto_broadcast` only works for binary pow. fixed